src/sentry/api/endpoints/organization_events.py

```diff
@@ -196,12 +204,64 @@ def get(self, request, organization):
             }, status=400)
 
         try:
-            tag_key = tagstore.get_group_tag_keys_and_top_values(
+            tag_keys = tagstore.get_group_tag_keys_and_top_values(
                 project_ids, None, environment_ids, keys=lookup_keys, get_excluded_tags=True, **snuba_args)
         except tagstore.TagKeyNotFound:
             raise ResourceDoesNotExist
 
-        return Response(serialize(tag_key, request.user))
+        if non_tag_lookup_keys:
+            tag_keys.update(self.handle_non_tags_keys(non_tag_lookup_keys, snuba_args))
```

Comment:
should this method be `non_tag_keys`? The double plural is hard to say. :bike:

Reply:
oh lol. thanks. :)